### PR TITLE
New tokens from production

### PR DESCRIPTION
**Pull request by AnaMontes11**

### CODE REVIEW

File: .github/workflows/schema-validator.yml [hunk 1]
```diff
@@ -16,14 +16,14 @@ jobs:
           file: "tokens/movistar.json"
           schema: "tokens/schema/skin-schema.json"
 
-  Movistar-classic:
+  Movistar-legacy:
```
Comment (yceballost) [inline .github/workflows/schema-validator.yml]: why this change is appearing when this already up in pre-production? :S @aweell 
https://github.com/Telefonica/mistica-design/blob/a2fd5e24cc7f1f80ef32e64d8f62bf0764b29098/.github/workflows/schema-validator.yml#L19

Comment (aweell) [reply .github/workflows/schema-validator.yml]: Pre-production should be merged into this branch first, then these changes that are already in pre will dissapear

**Review by aweell — APPROVED** [review]

File: .github/workflows/schema-validator.yml (hunk 1 above)
Comment (AnaMontes11) [reply .github/workflows/schema-validator.yml]: this is ready @yceballost

**Review by yceballost — APPROVED** [review]
